tools/main.c

```diff
@@ -60,7 +60,7 @@ main (int   argc,
     case 1:
         execute_man (argv[0], envp);
         LOG_ERR ("failed to load manpage, check your environment / PATH\n");
-        /* fallthrough */
+        exit (1);
```

Comment:
@pohly those comments comes from commit 9466263446fb ("build: use supported comment to suppress GCC7 fallthrough warning") that fixed the issue for me. AFAIK -Wimplicit-fallthrough is the same than -Wimplicit-fallthrough=3.

Now, looking at the [GCC docs](https://gcc.gnu.org/onlinedocs/gcc/Warning-Options.html), it seems that indeed -Wimplicit-fallthrough=2 doesn't match this comment since uses the following regular expression to match .*falls?[ \t-]*thr(ough|u).*

Which seems weird for me since I would had expected -Wimplicit-fallthrough=2  to be more relaxed than -Wimplicit-fallthrough=3

It's a pity that I didn't use fallsthrough that's matched for level 2 and 3, but again -Wimplicit-fallthrough=4 only matches fallthrough...